pnpm-workspace.yaml

```diff
@@ -2,7 +2,9 @@ packages:
   # all packages in direct subdirs of packages/
   - 'packages/ai-core'
   - 'packages/gen-ai-hub'
+  # sample code
+  - 'sample-code'
```

Comment:
```suggestion
  - '!sample-code'
```

So we don't publish it

Reply:
But wouldn't this also exclude the package from the build process?

Reply:
I think it would. I will take care of this on my canary release branch.